test/junit/scala/OptionTest.scala

```diff
@@ -0,0 +1,10 @@
+package scala
+
+import org.junit.Assert._
+import org.junit.Test
+
+class OptionTest {
+  @Test def testOptionZip1: Unit = assertEquals(Some("foo") zip List("bar"), Some(("foo", "bar")))
+  @Test def testOptionZip2: Unit = assertEquals(Some("foo") zip Nil, None)
+  @Test def testOptionZip3: Unit = assertEquals(None zip List("bar"), None)
```

Comment:
A test case with a collection of size > 1 as well?

Reply:
good idea